client/app/reducers/commentsReducer.js

```diff
@@ -20,12 +21,14 @@ export default function commentsReducer($$state = $$initialState, action = null)
       return $$state.merge({
         $$comments: comments,
         fetchCommentError: null,
+        isFetching: false,
```

Comment:
We also show an ajax indicator when saving the record.

If that is the case, why call the flag `isFetching`? A reference to the fetch API?

How about `ajaxBusy`?